### PR TITLE
Extended netboot_info + replaced hardcoded url

### LINKED ISSUE #364: do not hardcode package url in netboot information.
ideally we would not hardcode the url but obtain it from a pkg instance with  `{{ pkg.get_full_url }}`

So `netboot_info` has to be expanded and obtain the package and pass it to the template.

_Originally posted by @jelly in https://github.com/archlinux/archweb/pull/363#discussion_r676006244_

### PR DESCRIPTION
**Pull request by k4awon**
This should resolve #364. Extended netboot_info to get ipxe package and pass along to netboot template.

### CODE REVIEW

**Review by jelly — CHANGES_REQUESTED** [review]

File: releng/views.py [hunk 1]
```diff
@@ -90,7 +91,11 @@ def netboot_config(request):
 
 
 def netboot_info(request):
-    return render(request, "releng/netboot.html",
+    ipxepkg = Package.objects.get(pkgname='ipxe')
```
[the remark this answers is not in the document]
Comment (k4awon) [reply releng/views.py]: Thanks for the feedback. Added a try/except for error handling

[the remark this answers is not in the document]
Comment (k4awon) [reply releng/views.py]: Made a mistake here actually. Fixing this right now

[the remark this answers is not in the document]
Comment (k4awon) [reply releng/views.py]: Should be good now!

**Review by jelly — APPROVED** [review]
Thanks!

**Pull request merged**